content/09.evolution.md

```diff
@@ -72,6 +72,33 @@ This pattern of convergent evolution at some sites could indicate that certain m
 While it is evident that SARS-CoV-2 exhibits moderate potential for ongoing and future evolution, the relationship between mutations and pathogenicity is not yet known.
 Additional data is needed in order to understand patterns of evolutionary change and whether they are likely to affect virulence.
 
+Several factors could promote the evolution of SARS-CoV-2 including immunodeficiency and the transient exposure to antibodies directed against SARS-CoV-2 proteins. 
+A single case study of SARS-CoV-2 infection in an immunocompromised female with chronic lymphocytic leukemia (CLL) and hypogammaglobulinemia [@doi:10.1016/j.cell.2020.10.049] suggested that an accelerated evolution of the virus could occur in conditions of immunodeficiency. 
+A first administration of convalescent plasma did not clear the virus, and an ensuing increase in the genomic diversity in the samples was observed, suggesting an accelerated evolution due to selection pressure. 
+A second administration of convalescent plasma was able to clear the virus from the host 105 days after the initial diagnosis. However, during the whole infection time, the patient was asymptomatic but contagious.
+
+A second single case study in a 45-year old male with antiphospholipid syndrome [@doi:10.1056/NEJMc2031364], confirmed the earlier results of a persistent COVID-19 symptom in immunocompromised patients after 154 days of diagnosis leading to the death of patient. 
+The treatment included Remdesivir and the Regeneron anti-spike protein antibody cocktail. 
+Genomic analyses of the patient nasopharyngeal swabs confirmed an accelerated evolution of the virus through mutations in spike gene and the receptor-binding domain. 
+
+In summary, these two cases studies suggested an accelerated evolution and persistent shedding of the virus in conditions of immunodeficiency. 
+In particular, the first case highlighted the role of convalescent plasma in creating escape variants. 
+In fact, a study [@doi:10.1101/2020.12.28.424451] suggested that exposure to convalescent plasma in vitro could neutralize the virus after 7 days but after 45 days and up to 80 days, viral escape was promoted through mutations in the receptor-binding domain (RBD) and N-terminal domain (NTD). 
```

Comment:
Wow this is a super fascinating paper! I skimmed it and I had no idea this kind of research was even going on, how cool!
Looking at it, I think it was 6 passages/38 days of stable neutralization, and then the 7th passage was on the 45th day? 
This study is super different than anything else I've read for this paper! Maybe a little more background would help explain what they did? I am not super clear on what PT188 was selected but here is my rough attempt (but please feel free to edit/modify!):
```suggestion
In fact, one study [@doi:10.1101/2020.12.28.424451] exposed the SARS-CoV-2 virus to convalescent plasma _in vitro_ repeatedly to see how much plasma was required to neutralize the virus.
The results of the first six exposures were similar, but they reported that after the seventh exposure (on day 45), the amount of plasma required began to increase.
In analyzing the viral variants present, they found that this viral escape was promoted by the sudden accumulation of mutations, especially in the receptor-binding domain (RBD) and N-terminal domain (NTD), that quickly rose in frequency.
By the thirteenth exposure (day 85), the virus had evolved three mutations and could not longer be neutralized by the plasma used.
```

Reply:
yes pretty cool right? I think with the machine learning natural language processing tool, these are two of the most exciting papers I've read in a while.